packages/@lwc/rollup-plugin/src/index.ts

```diff
@@ -37,8 +39,11 @@ export interface RollupLwcOptions {
     enableScopedSlots?: boolean;
     /** The configuration to pass to `@lwc/compiler` to disable synthetic shadow support */
     disableSyntheticShadowSupport?: boolean;
+    apiVersion?: APIVersion;
```

Comment:
I'm assuming the `apiVersion` passed in to the `@lwc/rollup-plugin` will probably come from `lwc.config.json` or something (someday). It should be treated as a default that can be overridden by the `js-meta.xml` file.